c7n/resources/ebs.py

```diff
@@ -1180,11 +1180,14 @@ class CreateSnapshot(BaseAction):
                   - type: snapshot
                     copy-tags:
                       - Name
+                    tags:
```

Comment:
the tag docs should also be added to this action

Reply:
Done.  

The tags work a little differently here because `copy-volume-tags` works a little differently.  In `ec2.py`, `copy-volume-tags` uses a different parameter on the call to the API ('CopyTagsFromSource' instead of 'TagSpecifications') whereas in `ebs.py`, they both use 'TagSpecifications'.  I think the API call in `ec2.py` could send both parameters, so I could add `tags` to work for both `copy-volume-tags` and `copy-tags`, but I didn't know if there was a reason for that existing functionality, so I only added `tags` to `copy-tags` there.